include/odom.h

```diff
@@ -59,17 +59,17 @@ class Odom
 
   float getLinVelX() const
   {
-    return odom_vel_(1, 1);
+    return robot_vel_(1, 1);
```

Comment:
Can you elaborate a bit on this change.

Reply:
[nav_msgs/Odometry](http://docs.ros.org/en/noetic/api/nav_msgs/html/msg/Odometry.html) pose must be in the frame specified in `header.frame_id` and twist must be in `child_frame_id`. [robotont driver](https://github.com/robotont/robotont_driver/blob/noetic-devel/src/plugin_odom.cpp#L10) has the default `header.frame_id` as 'odom' and `child_frame_id` as 'base_footprint'. That means that while the pose data should be in the odom frame, the velocity (/twist) data needs to be in the 'base_footprint' frame, aka the robot frame. 

Previously this was not the case, as the velocity data was outputted in the odom frame. Fixing this lead to robot_localization being able to make use of the velocity data.

Reply:
Thanks for this fix and explanation. I believe I had missed the clearly commented lines at the beginning of the Odometry msg and intuitively thought they should be in the same frame defined by the header.frame_id.